_posts/2017-02-07-how-to-test-a-react-app-using-capybara-webkit.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title:  "How to test a React app using capybara-webkit"
+date: 2017-02-07 13:52:00
+categories: ["rails", "react"]
+author: "mauro-oto"
+---
+
+How to test a React app using capybara-webkit
+
+I have been using the [capybara-webkit](https://github.com/thoughtbot/capybara-webkit)
+gem for a while now since I first tried it out after making the switch from
+Capybara + Selenium.
+
+Using capybara-webkit speeds up my Selenium tests due to its headless nature,
+and it's very straight-forward to use. However, I had some trouble testing a
+Rails based React app.
+
+In this post, I will explain how I worked around the issues that came up when
+trying to use capybara-webkit with React.
+
+<!--more-->
+
+capybara-webkit depends on Qt, which you can install on OSX using [Homebrew](http://brewformulas.org/Qt).
+When running the tests using capybara-webkit with Qt 4.8, I noticed nothing
+rendered. To debug this problem, I used capybara-webkit's debug mode, which
```

Comment:
@mauro-oto I'd say "... nothing was getting rendered."

Maybe link "capybara-webkit's debug mode" to https://github.com/thoughtbot/capybara-webkit#configuration?